### ISSUE TITLE
Does it make a difference whether digitizers leave their hand inside the lightbox during imaging?

### ISSUE THREAD

**Issue by RebekkaML** [NHMDenmark/Pinned-Insects-workstation#69]
As discussed during the last digitization meeting (07.02.2024), Digitizers have a different approach as to whether they leave their hand inside the lightbox during imaging or not. This might change the lighting and needs to be tested.

@RebekkaML took test images for this on 09.02.2024 (Folder: 2024-02-09-RL on the PIOF01 Workstation computer). In the morning, I took my hand out of the lightbox, in the afternoon I left it in during imaging.

Without hand: img. 39_9415 - 39_9548 (123img)
With hand: img. 39_9549 - 39_9707 (159img)

For the first 10 specimens, I additionally took 2 images of each so they can be directly compared. Those images are separately stored in a folder called "2024-02-09_hand test" on the N-Drive: 

N:\SCI-SNM-DigitalCollections\DaSSCo\Workflows and workstations\Pinned_insects\2024-02-09_hand test

**Comment by RebekkaML** [comment]
I also uploaded all pictures from that day as JPG on the N-Drive now: N:\SCI-SNM-DigitalCollections\DaSSCo\Workflows and workstations\Pinned_insects\2024-02-09-RL-all-JPG

**Comment by chelseagraham** [comment]
Chelsea to assess and then ask Kim if there is a comprehensive tool that we can apply to assess / compare images

**Comment by chelseagraham** [comment]
Sampling the same points in Camera Raw for three sets of images from Rebekka's 2024-02-09_hand test folder to compare:
**A/**
9417 out vs. 9418 in
<img width="1204" alt="Screenshot 2024-04-12 at 11 18 50" src="https://github.com/NHMDenmark/Pinned-Insects-workstation/assets/108529653/5ecc19c6-df90-464f-998d-ad932994bab1">
<img width="1204" alt="Screenshot 2024-04-12 at 11 18 56" src="https://github.com/NHMDenmark/Pinned-Insects-workstation/assets/108529653/1f18d110-a696-4a7e-ba61-8d9217d5bfc7">
Comparing the values of the points sampled, the average difference of the average of RGB values was 0,85 units. The greatest difference in average value was between point 3, the top right corner, at 3 units. Points 5, 6, 8 had the smallest difference at 0 units. The most central point, point 9, had a difference of 0.67 units. The largest difference between RGB values was seen at point 3, where all RGB values were 3 units higher in the in image. 
**B/**
9423 out vs. 9424 in
<img width="1204" alt="Screenshot 2024-04-12 at 12 07 04" src="https://github.com/NHMDenmark/Pinned-Insects-workstation/assets/108529653/2af94d49-ee38-4da8-aeff-db7a2eaf9525">
<img width="1204" alt="Screenshot 2024-04-12 at 12 07 09" src="https://github.com/NHMDenmark/Pinned-Insects-workstation/assets/108529653/096a4b74-5b81-4c86-a8fe-c3987ce29a15">
Comparing the values of the points sampled, the average difference of the average of RGB values was 0,78 units. The greatest difference in average value was between point 3, the top right corner, at 3 units. Point 2 had the smallest difference at 0 units, followed by points 1, 6, and 4 which had a difference of ,33 units. The most central point, point 9, had a difference of 0.66 units. The largest difference between RGB values was seen at point 3, where all RGB values were 3 units higher in the in image. 
**C/**
9431 out vs. 9432 in
<img width="1204" alt="Screenshot 2024-04-12 at 12 37 44" src="https://github.com/NHMDenmark/Pinned-Insects-workstation/assets/108529653/b44b34ba-1c96-4321-ba6f-7a1fc2ea0f67">
<img width="1204" alt="Screenshot 2024-04-12 at 12 37 48" src="https://github.com/NHMDenmark/Pinned-Insects-workstation/assets/108529653/c846a1b0-41a5-4060-94f2-ff9b9e624ee5">
Comparing the values of the points sampled, the average difference of the average of RGB values was 0,81 units. The greatest difference in average value was between point 3, the top right corner, at 2 units. Point 4 had the smallest difference at 0 units, followed by points 7 and 8 which had a difference of ,33 units. The most central point, point 9, had a difference of 1 unit. The largest difference between RGB values was seen at point 3, where all RGB values were 2 units higher in the in image. 

In conclusion, these differences are negligible. Digitizers can work to their own comfort in removing their hands entirely or leaving them slightly in the lightbox. However, if a Digitizer is wearing very reflective jewelry, or notices a reflection in their images, they should remove the offending article.
